crescent/internal/handle_resp.py

```diff
@@ -152,7 +158,7 @@ def _get_command(
 }
 
 
-def _context_from_interaction_resp(interaction: CommandInteraction) -> Context:
+def _base_context_from_interaction_resp(interaction: CommandInteraction) -> BaseContext:
```

Comment:
I think this should be on the BaseContext as well, but that might be outside the scope of this PR.

Reply:
It has to be in this file because a few helper methods I feel should be defined here are defined here. (`_options_to_kwargs`, `_resolved_data_to_kwargs`)